app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyTextAreaQuestionScreen.kt

```diff
@@ -6,35 +6,42 @@ import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material.TextField
 import androidx.compose.material.TextFieldDefaults
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.focus.onFocusChanged
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import com.kks.nimblesurveyjetpackcompose.model.SurveyAnswer
 import com.kks.nimblesurveyjetpackcompose.ui.presentation.common.SurveyText
 import com.kks.nimblesurveyjetpackcompose.ui.theme.White20
+import kotlinx.coroutines.delay
+
+private const val DELAY_VALUE_CHANGE_MILLI = 300L
 
 @Composable
 fun SurveyTextAreaQuestionScreen(
     answers: List<SurveyAnswer>,
-    shortText: String,
-    onChooseAnswer: (answers: List<SurveyAnswer>) -> Unit
+    onAnswerChange: (answers: List<SurveyAnswer>) -> Unit
 ) {
-    var answer by remember { mutableStateOf("") }
+    var answer by remember { mutableStateOf(answers.first().answer) }
+
+    LaunchedEffect(key1 = answer) {
+        delay(DELAY_VALUE_CHANGE_MILLI)
+        onAnswerChange(answers.map { surveyAnswer -> surveyAnswer.copy(answer = answer, selected = true) })
+    }
 
     TextField(
         value = answer,
         onValueChange = { answer = it },
         shape = RoundedCornerShape(10.dp),
         placeholder = {
             SurveyText(
-                text = shortText,
+                text = answers.first().text,
```

Comment:
What if once in a blue moon `answers` doesn't have any element?
```suggestion
                text = answers.firstOrNull()?.text.orEmpty(),
```